src/components/transactions/TransactionItem.vue

```diff
@@ -73,7 +74,9 @@ export default {
     }
   },
   data: () => ({
-    show: false
+    show: false,
+    showValidators: false,
```

Comment:
why do we need this boolean?

Reply:
this is to make the show-all-validators thing in the WithdrawRewardsMessageDetails component independent from the `show` boolean coming from TransactionItem (so in the extension you actually get the cool effect of validators going vertical with their names)